.nomad/conf/.env.tpl

```diff
@@ -0,0 +1,28 @@
+{{- range $key, $value := (key "apiary/shared" | parseJSON) -}}
+{{- $key | trimSpace -}}={{- $value | toJSON }}
+{{ end -}}
+CAS_HOSTNAME="sso.gatech.edu"
+CAS_REAL_HOSTS="sso.gatech.edu"
+CAS_SESSION_NAME="__Host-cas_session"
+CAS_LOGOUT_URL="https://sso.gatech.edu/cas/logout"
+CAS_VERSION="3.0"
+CAS_ENABLE_SAML="false"
+{{- range service "mysql" }}
+DB_SOCKET="{{- index .ServiceMeta "socket" | trimSpace -}}"
+{{ end }}
+REDIS_CLIENT="predis"
+REDIS_SCHEME="unix"
+{{- range service "redis" }}
+REDIS_PATH="{{- index .ServiceMeta "socket" | trimSpace -}}"
+{{ end }}
+REDIS_PASSWORD="{{- key "redis/password" | trimSpace -}}"
+MAILGUN_DOMAIN="my.robojackets.org"
+MAILGUN_SECRET="{{- key "mailgun/secret" | trimSpace -}}"
+SESSION_SECURE_COOKIE="true"
+SESSION_COOKIE="__Host-apiary_session"
+{{ range $key, $value := (key (printf "apiary/%s" (slice (env "NOMAD_JOB_NAME") 7)) | parseJSON) -}}
+{{- $key | trimSpace -}}={{- $value | toJSON }}
+{{ end -}}
+APP_ENV="{{ slice (env "NOMAD_JOB_NAME") 7 }}"
+APP_URL="https://{{- with (key "nginx/hostnames" | parseJSON) -}}{{- index . (env "NOMAD_JOB_NAME") -}}{{- end -}}"
+HOME="/secrets/"
```

Comment:
I'm writing a MySQL config file to `/secrets/.my.cnf` with the app's database credentials, so I can run `mysql` in a Nomad web shell and get a MySQL prompt without any further configuration.

I'm setting `/secrets/` as the home directory because that's the easiest place to mount the files that MySQL will search. [[mysql docs](https://dev.mysql.com/doc/refman/8.0/en/option-files.html#:~:text=Table%C2%A04.2%C2%A0Option%20Files%20Read%20on%20Unix%20and%20Unix%2DLike%20Systems)]